PurchasesCoreSwift/Identity/IdentityManager.swift

```diff
@@ -18,19 +18,20 @@ import Foundation
 
     static let anonymousRegex = #"\$RCAnonymousID:([a-z0-9]{32})$"#
 
-    @objc public var maybeCurrentAppUserID: String? {
-        return deviceCache.cachedAppUserID
+    @objc public var currentAppUserID: String {
+        guard let appUserID = deviceCache.cachedAppUserID else {
+            fatalError(Strings.identity.null_currentappuserid)
```

Comment:
can we make a note somewhere to call this out in the release notes? don't want it to get lost amongst all the migration notes...

Reply:
we already do crash on non-nil `appUserID`: 
- we listen for changes to `NSUserDefaults`, and if the `appUserID` is removed, we hard crash
- if for some reason we didn't get the UserDefaults notification, the entire SDK already assumed that `appUserID` would be non-nil, and it'll either crash or worse (unknown behavior) if it's nil. 

so this isn't really new behavior, it's just that in obj-c, non-nullability is _assumed_ but not asserted (which we could and should have done), but swift is more strict with how this is handled. 

Note that, other than a developer deleting the SDK's entries for appUserID in userDefaults, the only other way to get to a nil `appUserID` is an SDK bug.